tardis/model/base.py

```diff
@@ -15,7 +15,7 @@
 )
 from tardis.io.config_validator import validate_dict
 from tardis.io.config_reader import Configuration
-from tardis.io.util import HDFWriterMixin
+from tardis.io.util import HDFWriterMixin, config_iteratation
```

Comment:
I don't think this iteration function is used any more.

Reply:
Sure, will remove unused, unnecessary imports

Reply:
Didn't remove the import as it was being used, renamed it correctly.